src/ensembl/src/content/home/Home.scss

```diff
@@ -1,44 +1,223 @@
 @import 'src/styles/common';
 
 .home {
-  padding-bottom: 80px;
+  display: grid;
+  grid-template-rows: minmax(550px, auto) 85px;
+}
 
-  h2 {
-    color: $grey;
-    font-size: 14px;
-    font-weight: $normal;
-  }
+.container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  background-color: $blue;
+  background-image: linear-gradient($black, $blue);
+  padding-top: 40px; 
+}
+
+.middleHomeWrapper {
   position: relative;
 }
 
-.search {
-  background: $light-grey;
-  box-shadow: 0 3px 5px $medium-light-grey;
-  min-height: 235px;
-  padding: 15px 60px;
+.middleHome {    
+  display: grid;
+  grid-template-rows: 38px 110px 100px; 
+  justify-content: center;
+  height: 248px;
```

Comment:
why?

Reply:
No idea again :) I think it was when i was playing with the middle content that i might have forgotten to remove it after we got a proper fix. There is a pattern here, things that might have worked before and after the proper fix, they dont make any difference and then i forgot to remove them. But good spot, i have removed it.